jobrunner/run.py

```diff
@@ -90,6 +90,7 @@ def handle_pending_job(job):
                 cleanup_job(job)
             except Exception:
                 mark_job_as_failed(job, "Internal error when starting job")
+                cleanup_job(job)
```

Comment:
Is it worth having some specific log output here to indicate we hit this point?

Reply:
"Internal error when starting job" _is_ exactly that log output (all job status messages end up in the log as well as being pushed back to job-server)